installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,76 @@
+
+* For user-provisioned infrastructure, xref:../../installing/installing-mirroring-installation-images.adoc#installing-mirroring-installation-images[you created a registry on your mirror host] and obtained the `imageContentSources` data for your version of {product-title}.
+
+[IMPORTANT]
+====
+The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the vSphere platform and the installation process of {product-title}. Use the user-provisioned infrastructure installation instructions as a guide; you are free to create the required resources through other methods.
+====
+
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-vsphere-infrastructure.adoc[leveloffset=+1]
+
+[id="choosing-an-method-to-install-ocp-on-vsphere"]
+== Choosing a method to install {product-title} on vSphere
+
+You can install {product-title} on vSphere by using installer-provisioned or user-provisioned infrastructure. The default installation type uses installer-provisioned infrastructure, where the installation program provisions the underlying infrastructure for the cluster. You can also install OpenShift Container Platform on infrastructure that you provide. If you do not use infrastructure that the installation program provisions, you must manage and maintain the cluster resources yourself.
+
+See the xref:../../architecture/architecture-installation.html#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
+
+=== Installer-provisioned infrastructure installation of {product-title} on vSphere
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Installing a cluster on vSphere]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with no customization.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc#installing-vsphere-installer-provisioned-customizations[Installing a cluster on vSphere with customizations]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with the default customization options.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations[Installing a cluster on vSphere with network customizations]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with network customization options. The installation program allows for some customization to be applied at the installation stage.
```

Comment:
You could provide some details about why you might want to customise the networking configuration. For example:

"You can customize your OpenShift Container Platform network configuration during installation, so that your cluster can coexist with your existing IP address allocations and adhere to your network requirements."

Additionally, you could minimise the first sentence after the bullet title. For example

"You can install {product-title} on installer-provisioned vSphere infrastructure, with network customizations."

Reply:
Done, thank you!